app/helpers/ExerciseConfig/Pipeline/Box/Box.php

```diff
@@ -77,6 +77,17 @@ public function __construct(BoxMeta $meta) {
    */
   public abstract function getCategory(): string;
 
+
+  /**
+   * Whether the box can be optimized (merged with another box with equal functionality).
+   * Boxes are optimizable by default, but specific boxes may prevent this behavior if necessary.
+   * @return bool
+   */
+  public function isOptimizable(): bool
+  {
```

Comment:
Well, I have to say that I am kind of against having 2 formatting styles in one file :-) which clashes together... All other methods have AFAIK opening braces on the same line as signature of the function...